### PR TITLE
Skip undeterministic integration test

### PR DESCRIPTION
**Pull request by facs95**
Closes: #XXX

## Description

Skipping tests until resolved to unblock CI

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: tests/integration_tests/test_grpc_only.py [hunk 1]
```diff
@@ -45,6 +45,7 @@ def grpc_eth_call(port: int, args: dict, chain_id=None, proposer_address=None):
     ).json()
 
 
+@pytest.mark.skip(reason="undeterministic test - https://github.com/evmos/ethermint/issues/1530")
```
Comment (fedekunze) [inline tests/integration_tests/test_grpc_only.py:48]: break line

Comment (MalteHerrmann) [reply tests/integration_tests/test_grpc_only.py:48]: Either break line or we could add a flake8 config file as we have recently decided on 120 characters max line length for the [proto files](https://github.com/evmos/ethermint/blob/main/.protolint.yml#L74)?

**Review by MalteHerrmann — APPROVED** [review]
LGTM

File: tests/integration_tests/test_grpc_only.py (hunk 1 above)
Comment (fedekunze) [reply tests/integration_tests/test_grpc_only.py:48]: @MalteHerrmann yes, let's add that

**Pull request merged**